flycheck.el

```diff
@@ -832,11 +832,11 @@ This variable is a normal hook.  See Info node `(elisp)Hooks'."
     (define-key map "e"         #'flycheck-set-checker-executable)
     (define-key map "?"         #'flycheck-describe-checker)
     (define-key map "h"         #'flycheck-display-error-at-point)
+    (define-key map "x"         #'flycheck-explain-error-at-point)
```

Comment:
I don't think that's required; just remove the separate binding in the docs.  It's ok if it's only available under `M-x`.